### PR TITLE
Mod 9201 homepage link to about the data

### PR DESCRIPTION
**Pull request by chasls2**
**High level description:**

In the Resources section, change the third item (currently “Data Model”) to:

Title: About the Data

Subtitle: Read important data disclosures

Link: View the disclosures [opens up About the Data module]

Change “our data” to “the data”

in “Analyst Guide” section

in “Data Dictionary” section

**Technical details:**
Updated the Data Model to say About the Data

Changed our data to the data in Analyst Guide and Data Dictionary 

**JIRA Ticket:**
[DEV-9201](https://federal-spending-transparency.atlassian.net/browse/DEV-9201)

**Mockup:**
no mocks

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/js/components/homepageUpdate/HomepageResources/HomepageResources.jsx [hunk 1]
```diff
@@ -60,11 +60,11 @@ const cardObjects = [
...
             </div>
         ),
-        headline: 'Data Model',
-        text: 'Learn how our data is organized',
+        headline: 'About the Data',
+        text: 'Read important data disclosures',
         buttonText: (
             <>
-                View the model&nbsp;&nbsp;
+                View the disclosures;
```
Comment (brianpetway) [inline src/js/components/homepageUpdate/HomepageResources/HomepageResources.jsx:67]: the two &nbsp need to be here for spacing between the word and the arrow

File: src/js/components/homepageUpdate/HomepageResources/HomepageResources.jsx [hunk 2]
```diff
@@ -60,11 +60,11 @@ const cardObjects = [
...
                 <FontAwesomeIcon icon="arrow-right" />
             </>
         ),
```
Comment (brianpetway) [inline src/js/components/homepageUpdate/HomepageResources/HomepageResources.jsx:70]: at line 71, the link needs to change so this click will open the new About the Data module

Comment (chasls2) [reply src/js/components/homepageUpdate/HomepageResources/HomepageResources.jsx:70]: fixed

File: src/js/components/homepageUpdate/HomepageResources/HomepageResources.jsx (hunk 1 above)
Comment (chasls2) [reply src/js/components/homepageUpdate/HomepageResources/HomepageResources.jsx]: fixed

File: src/js/components/homepageUpdate/HomepageResources/HomepageResources.jsx (hunk 2 above)
Comment (chasls2) [reply src/js/components/homepageUpdate/HomepageResources/HomepageResources.jsx:70]: trying to find the correct link now

Comment (chasls2) [reply src/js/components/homepageUpdate/HomepageResources/HomepageResources.jsx:70]: just to make sure is the the correct link
https://www.usaspending.gov/data-sources

Comment (chasls2) [reply src/js/components/homepageUpdate/HomepageResources/HomepageResources.jsx:70]: oh man... I knew that.. so dumb..
Ill see if I can link it

**Review by brianpetway — APPROVED** [review]

**Pull request merged**